docker-compose.yml

```diff
@@ -1,4 +1,8 @@
-x-env_file: &env_file .env.development
+x-env_file: &env_file
+  - path: .env.development
+    required: true
+  - path: .env.local
+    required: false
```

Comment:
This is in more recent versions of docker compose. It allows us to assume an `.env.local` exists.